manage.py

```diff
@@ -32,6 +32,37 @@
 manager.add_command('runserver', Server(use_debugger=True, use_reloader=True))
 
 
+def check_itemized_queues(schedule):
+    """Checks to see if the queues associated with an itemized schedule have
+    been successfully cleared out and sends the information to the logs.
+    """
+
+    remaining_new_queue = db.engine.execute(
+        'select count(*) from ofec_sched_{schedule}_queue_new'.format(
+            schedule=schedule
+        )
+    ).scalar()
+    remaining_old_queue = db.engine.execute(
+        'select count(*) from ofec_sched_{schedule}_queue_old'.format(
+            schedule=schedule
+        )
+    ).scalar()
+
+    if remaining_new_queue == remaining_old_queue == 0:
```

Comment:
@LindsayYoung: I've been thinking about this more throughout the day and I wonder if we even want to do this.  The reason I question this now is that we're only clearing the records that we've processed for Schedule A and B*, not the entire queue.  Presumably at the end the queues would be clear, but theoretically speaking records could still be added to the queue during processing as you noted above.

I wonder if this is why we've seen some issues with the Schedule E records being out of sync - the queue processing takes place, but records are being added/modified/deleted while the processing is taking place, so at the end when the queues are cleared, we lose what was added to them during that time.

Would it just make more sense to focus on removing the records that we know we processed (or even as we process them - that might be too intensive and slow, though)?  If so, then there wouldn't be a need for this check anymore.

Alternatively, I wonder if we could modify the stored procedure to just take care of this entirely for us each time a record comes through.  It could be intensive at times, but if we have a read replica in place then only the master would be dealing with it.  When the replica is refreshed with the nightly backup it'll get the new changes, so the update cadence would remain the same as it does now.

*The current implementation in this PR deletes all records in the Schedule A and B queues based on the cycle we're currently in.  Perhaps that should strictly be the list of `sub_ids` that we successfully update instead.